Multi ATIS Support Pages (#163)
vatnz-dev/sops: docs/controller-skills/Client-Guides/vatSys-setup.md

Our vatSys dataset currently ships with two plugins, the VATNZ Standard Route Tool and the Multiple ATIS Editor.

### VATNZ Standard Route Tool

The VATNZ Standard Route Tool allows controllers to quickly look up standard routes that aircraft have filed, as well as issue pilots with a standard route on request.

!!! info "What is a Standard Route?"
    A Standard Route is a predefined route that is issued to IFR aircraft around New Zealand. These routes are made up of defined nav aids, waypoints, and airways that link to each other, and provide a safe and standard way for IFR aircraft to fly around the country.

    The standard IFR clearance in New Zealand is issued as an "airways clearance". Examples of standard routes can be found [here](https://www.aip.net.nz/assets/AIP/Aerodrome-Charts/Auckland-NZAA/NZAA_61.1.pdf){:target="blank"}

You can find this plugin under `Tools > Standard Route Tool`

<figure markdown>
  ![vatSys Tools Menu](../assets/vatSys-tools.png) 
  <figcaption>vatSys Tools Menu</figcaption>
</figure>

A full user manual for this plugin can be found [**here**](https://www.vatnz.net/vatnz/resources/standard-route-tool/){:target="blank"}

### Multiple ATIS Editor

> The Multiple ATIS Editor replaces the default vatSys ATIS editor and allows controllers to have up to four (4) ATIS connections whilst controlling.

Reviewer: Not sure the significance of putting both four and 4?

Author: Removed.

<figure markdown>
  ![vatSys Tools Menu](../assets/vatSys-multi-atis-menu.png) 
  <figcaption>vatSys Multiple ATIS Editor</figcaption>
</figure>

> This plugin follows the same color scheme as our domestic dataset and also has the same functions as the vatSys ATIS editor.

Reviewer: ```suggestion
This plugin follows the same colour scheme as our domestic dataset and also has the same functions as the vatSys ATIS editor.
```

> Fields in yellow have changed and need to be reviewed and updated.

Reviewer 2: ```suggestion
When a new METAR is issued, fields highlighted in yellow indicate a change that needs to be reviewed and updated.
```

Reviewer: Probably worth also pointing that, from memory, the issued at time doesnt highlighted and needs altering manually?

<figure markdown>
  ![vatSys Tools Menu](../assets/vatSys-multi-atis-menu1.png) 
  <figcaption>Updated Fields</figcaption>
</figure>

The new ATIS editor can be found under `Windows > ATIS Editor`

<figure markdown>
  ![vatSys Tools Menu](../assets/vatSys-atis-menu.png) 

>   <figcaption>Updated Fields</figcaption>

Reviewer 2: ```suggestion
  <figcaption>vatSys Windows Menu</figcaption>
```
